README.rst

```diff
@@ -64,7 +64,7 @@ the executable is called ``python.exe``; elsewhere it's just ``python``.
 Building a complete Python installation requires the use of various
 additional third-party libraries, depending on your build platform and
 configure options.  Not all standard library modules are buildable or
-useable on all platforms.  Refer to the
+usable on all platforms.  Refer to the
```

Comment:
Looks like a difference between UK and American English, not exactly a typo. Let's just keep it the old way.

```suggestion
useable on all platforms.  Refer to the
```